php/ext/google/protobuf/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+
+# Protobuf PHP extension (PECL)
+load("@rules_pkg//:mappings.bzl", "pkg_files", "pkg_filegroup", "strip_prefix")
+load("//:protobuf_version.bzl", "PROTOBUF_PHP_VERSION")
+
+filegroup(
+  name = "template_package_xml",
+  srcs = ["template_package.xml"],
+)
+
+pkg_files(
+  name = "utf8_range_files",
+  srcs = [
+    "@utf8_range//:utf8_range_srcs",
+    "@utf8_range//:LICENSE",
+  ],
+)
+
+pkg_files(
+  name = "source_files",
+  srcs = glob([
+    "*.h",
+    "*.c",
+  ]) + [
+    "LICENSE",
```

Comment:
There isn't a LICENSE file in this folder, I think you need to use //:LICENSE

Reply:
Previously, this was attempting to maintain compatibility with the release script which copies LICENSE (and third_party/utf8_range) over. However, since third_party/utf8_range is now a submodule this is broken now anyways.

Replaced with LICENSE and updated release script to use files in original location, but respect the directory strucure specified by pkg_files